clang/docs/SYCLMainPageforRFC.md

```diff
@@ -0,0 +1,28 @@
+# Add full support for the SYCL programming model
+
+We (Intel) would like to request to add full support for SYCL programming model support to LLVM/Clang project to facilitate collaboration on C++ single-source heterogeneous programming for accelerators like GPU, FPGA, DSP, etc. from different hardware and software vendors. The SYCL 2020 Specification is available at the Khronos site: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html.
+
+Our prior RFC toward this effort can be seen here: https://lists.llvm.org/pipermail/cfe-dev/2019-January/060811.html
+
+In the four years since then: 
+  *   both our design and SYCL have evolved (e.g. the previous RFC supported SYCL 1.2.1)
+  *   we have multiple years of SYCL implementation experience
+  *   we have received feedback from the community for both the original RFC as well as patches that we have upstreamed
+  *   we have real-world user feedback from talking with our customers
+  *   we have worked with Codeplay in supporting non-Intel hardware
+and as a result now have a more mature set of design and implementation that we can upstream that is more in line with community standards and expectations.
+
+SYCL is an open specification and Intel wants an implementation that fully meets this specification in the community.  This allows for open tooling and is in line with the goals of the Unified Acceleration (UXL) Foundation: https://www.oneapi.io/blog/announcing-the-unified-acceleration-uxl-foundation/
+
+We have long term commitment to supporting SYCL.  Alexey Bader, who is deeply involved with both the community and SYCL, is the SYCL code owner.
```

Comment:
I feel like we need to say more here than just dropping Alexey's name. Intel employs quite a few people working on SYCL. Can we say something about sponsorships? The UXL? Funding for The Khronos Group? Codeplay? Other contributors to LLVM and Clang?

```suggestion
We have a long term commitment to supporting SYCL.  Alexey Bader is the SYCL code owner and has a long history of work within the LLVM community and the development of the SYCL specification.
```

Reply:
Rephrased as you suggested.
But for the bigger point: @jbrodman, could you please suggest the necessary stuff here including anything we need for community/political correctness?